html/src/Post.js

```diff
@@ -116,7 +118,13 @@ class Post extends React.Component {
         />
       {stars}
       <div className="text-container read-more-container">
-        <div className="text read-section">
+        <div className={textClasses}
+            onClick={
+              () => {
+                sendMetric('post_expanded', {post_id: this.props.post.id});
```

Comment:
Can you put this in a single helper function: this.expand, or something, and then change the two locations this is called to use the same function.

You can make it an instance method of the class.